encrypted-media-respec.html

```diff
@@ -606,7 +606,83 @@
           <section>
             <h5>Get Supported Configuration</h5>
             <p>Given a <a def-id="keysystems"></a> implementation <var>implementation</var>, <a>MediaKeySystemConfiguration</a> <var>candidate configuration</var>, and <var>origin</var>, this algorithm returns a supported configuration or <code>null</code> as appropriate.</p>
-            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.</p>
+            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.
+            </p>
+            <ol>
+              <li>
+                <dl class="switch">
+                  <dt>
+                    If the <a def-id="option-distinctiveIdentifier"></a> member of
```

Comment:
I worry about using `distinctiveIdentifier` as the (only) variant. For example, in #96, we want to account for other potential reasons. Perhaps we should use something like "has consent been denied", "has consent been requested", or "can request consent".

Then, anywhere that consent might be required (including by choice of the user agent), we could check this value and choose not to do whatever would require consent.

I don't think this materially changes the behavior. Passing a separate Boolean also allows us to avoid copying and modifying the configuration.